src/js/utils/ui.js

```diff
@@ -38,7 +38,7 @@ export default class UI extends Eventable {
         this.pointerId = null;
         this.startX = 0;
         this.startY = 0;
-        this.noFocusHelper = flagNoFocus(element);
+        this.noFocusHelper = flagNoFocus(element, options.focusOnMouseDown);
```

Comment:
@pajong Why is this it's own module and why is it being added in the constructor? `ui` was refactored recently with the design principal that it should only add listeners when needed.

This is not something that should be added in the constructor. The focus logic should only apply to ui instances handling mouse (and maybe touch) input in `eventRegisters`.